src/components/Review/ChangeRequests.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { MouseEvent } from 'react';
+import "./ChangeRequests.css";
+import ChangeRequest, { ChangeRequestElement } from "./ChangeRequest";
+
+import ClearIcon from '@material-ui/icons/Clear';
+import Button from '@material-ui/core/Button';
+
+export default function ChangeRequests({hash, reviewStatus}: {hash: string, reviewStatus: string}) {
```

Comment:
The same `ChangeRequest` and `ChangeRequests` are really confusing. Consider giving them more descriptive names.